tex/gregoriotex.lua

```diff
@@ -267,9 +267,23 @@ local function get_gregorioversion()
   return internalversion
 end
 
+local function map_fonts()
+  -- use greciliae because it is the most complete font
+  local font_info = fontloader.open(kpse.find_file('greciliae', 'truetype fonts'))
```

Comment:
I have to understand the stuff in luaotfload better.  For instance:

`fonts.names.resolve_cache({name = 'greciliae'}) returns `'greciliae.ttf'`, `false` (from `luaotfload-lookup-cache.lua` in `TEXMFVAR`)

I'll see what I can figure out.